web/views/manifestation.py

```diff
@@ -0,0 +1,64 @@
+from flask import Blueprint
+from flask_restful import reqparse, Resource, Api
+
+from coalaip import CoalaIp
+from coalaip_bigchaindb.plugin import Plugin
+from web.utils import get_bigchaindb_api_url
+
+
+coalaip = CoalaIp(Plugin(get_bigchaindb_api_url()))
+
+manifestation_views = Blueprint('manifestation_views', __name__)
+manifestation_api = Api(manifestation_views)
+
+
+def parse_model(required_fields):
+    def _parse_model(inputs):
+        for field in required_fields:
+            try:
+                value = inputs[field]
+            except KeyError:
+                raise KeyError('`{}` must be provided'.format(field))
+            if bool(value) is not True:
+                raise ValueError("`{}`'s value must be defined")
+        return inputs
+    return _parse_model
+
+
+class ManifestationApi(Resource):
+    def post(self):
+        manifestation_model = parse_model(['name', 'datePublished', 'url'])
+        work_model = parse_model(['name', 'author'])
+        user_model = parse_model(['verifyingKey', 'signingKey'])
+
+        parser = reqparse.RequestParser()
+        parser.add_argument('manifestation', type=manifestation_model,
+                            required=True, location='json')
+        parser.add_argument('work', type=work_model, required=True,
+                            location='json')
+        parser.add_argument('user', type=user_model, required=True,
+                            location='json')
+        args = parser.parse_args()
```

Comment:
looks awesome, just curious, if `parse_args()` throws `KeyError` or `ValueError`, does this yield a `400+message` or a `500`?